### PR TITLE
Add fedmenu to fedocal

### PR DESCRIPTION
**Pull request by pypingou**

### CODE REVIEW

**Comment by oddshocks** [comment]
:+1:

File: fedocal/templates/default/master.html [hunk 1]
```diff
@@ -114,6 +114,19 @@ <h4 class="hide">{{ _('Main Site Links') }}</h4>
             });
         });
     </script>
+
+    {% if config['FEDMENU_URL'] is defined %}
```
Comment (ralphbean) [inline fedocal/templates/default/master.html]: How does ``is defined`` work here?  I thought ``is defined`` was a jinja test to see if a var was defined.. but, ``config['FEDMENU_URL']`` is more than just a simple var.

Comment (pypingou) [inline fedocal/templates/default/master.html]: We can do without it indeed

**Comment by pypingou** [comment]
Jinja catches the exception and behaves as if it wasn't there :)

**Comment by ralphbean** [comment]
:tophat: magic! :tophat: 

:+1: to merge from me

**Comment by pypingou** [comment]
Thanks for the reviews :)

**Pull request merged**